dashboards/go/go-runtime-view-prometheus.json

```diff
@@ -0,0 +1,626 @@
+              "scale": "LINEAR"
+            },
+            "chartOptions": {
+              "mode": "COLOR",
+              "showLegend": false,
+              "displayHorizontal": false
+            }
+          },
+          "id": ""
+        }
+      },
+      {
+        "xPos": 24,
+        "width": 24,
+        "height": 10,
+        "widget": {
+          "title": "Job's Latest Versions (latest)",
+          "timeSeriesTable": {
+            "columnSettings": [
+              {
+                "column": "version",
+                "visible": true
+              },
+              {
+                "column": "value",
+                "visible": true
+              },
+              {
+                "column": "job",
+                "visible": true
+              }
+            ],
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "outputFullDuration": true,
+                  "prometheusQuery": "sum by(project_id, location, cluster, job, namespace, instance, version)(go_info{job=~\"${job.value}\", namespace=~\"${namespace.value}\", instance=~\"${instance.value}\"})",
+                  "unitOverride": "1"
+                }
+              }
+            ],
+            "displayColumnType": false,
+            "metricVisualization": "BAR",
+            "opsAnalyticsSettings": {
+              "maxRows": "0",
+              "pageSize": "0",
+              "showFilterBar": false
+            }
+          }
+        }
+      },
+      {
+        "width": 24,
+        "height": 10,
+        "widget": {
+          "title": "Instances by Version",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "sum by(project_id, location, cluster, job, namespace, instance, version) (go_info{job=~\"${job.value}\", namespace=~\"${namespace.value}\", instance=~\"${instance.value}\"})",
+                  "unitOverride": "1"
+                }
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
+      },
+      {
+        "width": 48,
+        "height": 10,
+        "widget": {
+          "title": "Version",
+          "collapsibleGroup": {
+            "collapsed": false
+          },
+          "id": ""
+        }
+      },
+      {
+        "yPos": 18,
+        "width": 48,
+        "height": 12,
+        "widget": {
+          "title": "Concurrency",
+          "collapsibleGroup": {
+            "collapsed": false
+          },
+          "id": ""
+        }
+      },
+      {
+        "yPos": 18,
+        "width": 24,
+        "height": 12,
+        "widget": {
+          "title": "Number of goroutines",
```

Comment:
Nit: Let's do a standardization pass on whether you want title case ("Number of Goroutines" / "Goroutines Count") or sentence case ("Number of goroutines").

I don't think one is preferred over the other as long as we are consistent across all the chart titles